include/toml++/impl/print_to_stream.inl

```diff
@@ -419,6 +419,8 @@ TOML_IMPL_NAMESPACE_START
 	void TOML_CALLCONV print_to_stream(std::ostream & stream, const source_region& val)
 	{
 		print_to_stream(stream, val.begin);
+		print_to_stream(stream, " to "sv);
+		print_to_stream(stream, val.end);
```

Comment:
There are situations wherein a source region can span zero characters, so it might be useful to take this into account:

```suggestion
		if (val.begin != val.end)
		{
			print_to_stream(stream, " to "sv);
			print_to_stream(stream, val.end);
		}
```